core/templates/dev/head/pages/exploration_player/SupplementalCardDirective.js

```diff
@@ -72,6 +72,12 @@ oppia.directive('supplementalCard', [function() {
         };
 
         $scope.submitAnswer = function(answer, interactionRulesService) {
+          // Do not clear the help card or submit an answer if there is an
```

Comment:
Not entirely sure what you mean by the question, but there's an analogue to this in $scope.submitAnswer in the conversation skin directive. I think the check got lost in the refactor to a separate supplemental card.